references/js-callout/test/integration/features/js-callout.feature

```diff
@@ -0,0 +1,8 @@
+Feature:
+  As the xyz service
+  I want to hide sensitive data from the response body
+  So that I expose only what is necessary for the clients
+
```

Comment:
2 space tab in `Feature:` and 4 in `Scenario:` - shall we make consistent?

Reply:
We have some feature files like this and others with no spacing. Looking at behat, panda and others, it looks like 2/4 spaces are the most used. vscode also does this type of styling with the cucumber plugin. So I will change the rest to this style.